### PR TITLE
Move roadmap into its own file and extend it

### PR DESCRIPTION
**Pull request by fabxc**
@brancz

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+of this service.
+
+Currently Prometheus only allows to configure Alertmanager URLs via flags
+on startup. Thus the Prometheus pods have to be restarted manually if the 
+endpoints change.
+PetSets or manually maintained headless services in Kubernetes allow to
+provide stable URLs working around this. In the future, Prometheus will allow
+for dynamic service discovery of Alertmanagers ([tracking issue](https://github.com/prometheus/prometheus/issues/2057)). 
+
+### Cluster-wide version
```
Comment (brancz) [inline ROADMAP.md]: this is outdated, and will only be updated by external systems and the operator performs updates based on changes to single Prometheus objects

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+doesn't exist yet. It is left to the user to populate it with the desired rules.
+
+It is still up for discussion whether it should be possible to include rule files living
+in arbitrary ConfigMaps by their labels.
+Intuitively, it seems fitting to define in each `ServiceMonitor` which rule files (based 
+label selections over ConfigMaps) should be deployed with it.
+However, rules act upon all metrics in a Prometheus server. Hence, defining the
+relationship in each `ServiceMonitor` may cause undesired interference.
+ 
+### Alerting
```
Comment (brancz) [inline ROADMAP.md]: This is already available :tada:

**Comment by fabxc** [comment]
Done

**Comment by brancz** [comment]
What about the "Alertmanager Roadmap" that I mentioned above? ([this](https://github.com/coreos/prometheus-operator/blob/0c180ebb0b4e77d311151a870ed4513d5817c368/Documentation/alertmanager.md#current-state-and-roadmap))

**Comment by fabxc** [comment]
Damnit, sorry. Now.

**Comment by brancz** [comment]
lgtm

**Pull request merged**